test/new_relic/agent/custom_event_aggregator_test.rb

```diff
@@ -97,6 +97,26 @@ def test_truncates_attributes
       assert_equal(expected, actual)
     end
 
+    def test_does_not_truncates_llm_embedding_input_attribute
+      params = {'input' => 'a' * 5000}
+      expected = {'input' => 'a' * 5000}
```

Comment:
If you ever want to test going over a maximum threshold, you could always stub the real constant with a lower value. That way you can just test the logic without needing to actually have strings that are crazy long to test with.

In this case...

```ruby
require 'minitest/stub_const'

def test_does_not_truncates_llm_embedding_input_attribute
  NewRelic::Agent::CustomEventAggregator.stub_const(MAX_ATTRIBUTE_SIZE, 5) do
    # ...
  end
end
```

Reply:
For me, I like to avoid stubbing for things that aren't too difficult to test directly like this. Since making a long string is pretty simple with ruby, I'd prefer testing the actual constant just in case it had been changed for some reason, because then we'd see the failure.

But when things get more complicated, like creating 10,000 spans or something, then I think this is a great way to go.

Reply:
OOooo I never thought about the possibility of stubbing very large values. I also kinda like testing directly here, but will keep in mind that practice for the future.

Reply:
Sounds good. I definitely support @tannalynn in her cautioning against the hiding of bugs that can result from a reliance on stubbing. In cases like this one, the constant predates the PR and the PR doesn't deliver anything that impacts how the constant is being evaluated. I was relying on there already being great, non-stubbed test coverage in place. With everything that is being tested in this PR being downstream of the constant having been checked, I think stubbing is ok. Not stubbing is ok too of course.